examples/docs/src/pages/features/latex.mdx

```diff
@@ -21,7 +25,14 @@ For example, the following code
 $\sqrt{a^2 + b^2}$
 ```
 
-will be rendered as: $\sqrt{a^2 + b^2}$
+will be rendered as: $\sqrt{a^2 + b^2}$.
+
+You can also do display math:
+```math
+\int_2^3x^3\,\mathrm{d}x
```

Comment:
I would prefer to add an example of mathjax usage in root/docs directory (source code of Nextra’s docs). These examples folder is just a playground